requirements/base.txt

```diff
@@ -6,13 +6,13 @@
 git+https://github.com/edx/XBlock.git@xblock-0.4.10#egg=XBlock==0.4.10
 
 # edx-submissions
-git+https://github.com/edx/edx-submissions.git@1.1.0#egg=edx-submissions==1.1.0
+git+https://github.com/edx/edx-submissions.git@efischer/gemnasium#egg=edx-submissions==1.1.0
```

Comment:
Note that I have the branch specified for testing purposes, it'll be a proper version bump once merged.

See https://github.com/edx/edx-submissions/pull/46